src/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare namespace snapshot {
+  // This is necessary, or TS will consider this to be a "non-module entity"
+}
+declare function snapshot(value: any): void;
```

Comment:
- it is not just `any` value, what I really would love to see is `any` value but not `null` or `undefined`, if that can be done with TypeScript
- can you also add overrides because there are named snapshots: https://github.com/bahmutov/snap-shot-it#named-snapshots and even data-driven snapshots with `function, ...any`